utils/keywords.py

```diff
@@ -0,0 +1,53 @@
+import pke 
+def search_keywords(text):
+    """ Extracts keywords from the text using TextRank algorithm.
+    Args:
+        text (str): Text from which keywords are to be extracted.
+    Returns:
+        keyphrases (list): List of keyphrases extracted from the text.
+        keywords (list): List of keywords extracted from the text.
+        """
+    # define the set of valid Part-of-Speeches
+    pos = {'NOUN', 'PROPN', 'ADJ'}
+
+    # 1. create a TextRank extractor.
+    extractor = pke.unsupervised.TextRank()
+
+    # 2. load the content of the document.
+    extractor.load_document(input=text,
```

Comment:
```suggestion
    extractor.load_document(input=text,
```
In my opinion there's no need to comment what you do in code, when it clearly says what it does.